### PR TITLE
Show,edit profile

### PR DESCRIPTION
**Pull request by HungNguyenKhanh**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -4,8 +4,14 @@
...
 
-    <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
-    <%= javascript_include_tag "application", "data-turbolinks-track": "reload" %>
+    <%= stylesheet_link_tag "application", media: "all",
+      "data-turbolinks-track": "reload" %>
+    <%= javascript_include_tag "application",
+      "data-turbolinks-track": "reload" %>
+    <link rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.2.0/css/all.css"
+      integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ"
```
Comment (thangtx88) [inline app/views/layouts/application.html.erb]: cái này là cái gì hả em

Comment (HungNguyenKhanh) [reply app/views/layouts/application.html.erb]: Em sử dụng icon từ https://fontawesome.com/

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -27,6 +34,10 @@ def user_params
   def get_user
     @user = User.find_by id: params[:id]
     return if @user
-    redirect_to :signup
+    redirect_to root_url
```
Comment (TranBaVinhSon) [inline app/controllers/users_controller.rb]: add `alert`

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (TranBaVinhSon) [reply app/views/layouts/application.html.erb]: @HungNguyenKhanh use `gem`

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -1 +1,53 @@
...
+          <div>
+            <div id="user-media-container" class="space-2">
+              <%= image_tag avatar_url(@user), class: "avatar-full" %>
+            </div>
+          </div>
+        </div>
+        <div class="panel space-4">
+            <div class="panel-header">Verified Info</div>
+            <div class="panel-body">
+              <ul class="list-unstyled space-3" title="Verified Info">
```
Comment (TranBaVinhSon) [inline app/views/users/show.html.erb]: why do you need `ul` without `li` ? please remove!

File: app/views/users/edit.html.erb [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+            <li class="sidebar-item">
+              <%= link_to t("edit_profile"), edit_user_path,
+                {class: "sidenav-item"}%>
+            </li>
+            <li class="sidebar-item">
+              <%= link_to t("verification"), edit_user_path,
+                {class: "sidenav-item"}%>
+            </li>
+            <li class="sidebar-item">
+              <%= link_to t("reviews"), edit_user_path,
```
Comment (TranBaVinhSon) [inline app/views/users/edit.html.erb]: Please remove `edit_user_path` and replace by `javascript:void(0)`. You will update update link in next pulls

File: app/views/users/edit.html.erb [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    <div class="row">
+      <div class="col-md-3 space-sm-4">
+        <div class="sidenav">
+          <ul class="sidebar-list">
+            <li class="sidebar-item">
+              <%= link_to t("edit_profile"), edit_user_path,
+                {class: "sidenav-item"}%>
+            </li>
+            <li class="sidebar-item">
+              <%= link_to t("verification"), edit_user_path,
```
Comment (TranBaVinhSon) [inline app/views/users/edit.html.erb]: Please remove `edit_user_path` and replace by `javascript:void(0)`. You will update update link in next pulls

**Pull request merged**